### PR TITLE
fixed the bug that bounds with value zero are omited and added a test…

### PR DESCRIPTION
**Pull request by typohnebild**
… for that

TIL: perl evaluates "0" to false. 
This lead that bounds with value 0 were omitted. Fixed that

### CODE REVIEW

File: lib/LaTeXML/Package/mws.sty.ltxml [hunk 1]
```diff
@@ -70,18 +70,18 @@ DefMathML('Apply:?:range',
         # get it
         my $low = $_[1]->textContent;
         my $high = $_[2]->textContent;
-        if($low && $high){
```
Comment (dginev) [inline lib/LaTeXML/Package/mws.sty.ltxml]: the cleaner way to do this in perl is
```perl
if (defined $low && defined $high)
```

you wouldn't have to change any of the other lines then

File: lib/LaTeXML/Package/mws.sty.ltxml [hunk 2]
```diff
@@ -70,18 +70,18 @@ DefMathML('Apply:?:range',
...
-            return ['m:csymbol', { cd => 'mws', name => 'range', low=>"$low", high=>"$high"}, 'range'];
+        my $attr = {
+         cd => 'mws',
+         name => 'range'
+        };
+        if($low eq 0 || $low){
+            $attr->{'low'} = $low;
         }
-        elsif($low){
```
Comment (dginev) [inline lib/LaTeXML/Package/mws.sty.ltxml]: and respectively `elsif(defined $low)` , $elsif(defined $high)`

**Comment by typohnebild** [comment]
Ah cool, i never came in touch with the defined in perl, thank you. But it is not exactly what i wanted, since i wanted also avoid that a low/high attribute is added when the string is empty. 

Do you have an idea why the test for the perl versions 5.14-5.20 fail?

**Comment by dginev** [comment]
@typohnebild Travis failure I think? If you rerun them they should pass, it looks like for some reason latexml failed to install.

There is another trick if you are ok with "any non-empty string" which is to test `if(length($low))` which will only pass if `$low` has any content, otherwise length would return 0 and fail.

**Comment by typohnebild** [comment]
@dginev Thanks for hint.
Yeah the travis test, they are still failing for the perl versions 5.20 - 5.14

**Comment by dginev** [comment]
Well, it's pretty clear that those versions can't install latexml as a dependency:
```
Warning: prerequisite LaTeXML 0.8.0 not found.
```

Two suggestions:
 - update the dependency to `0.8.4` in the Makefile.PL
 - just remove the old perl versions in Travis CI.

**Comment by dginev** [comment]
Good to merge :rocket:

**Pull request merged**